duktape/src/main/jni/DuktapeContext.cpp

```diff
@@ -165,7 +166,11 @@ void DuktapeContext::bindInstance(JNIEnv *env, jstring instance, jobject object,
     duk_push_pointer(m_context, javaMethod);
     duk_put_prop_string(m_context, func, JAVA_METHOD_PROP_NAME);
 
-    duk_put_prop_string(m_context, objIndex, javaMethod->getName());
```

Comment:
The name was only being used here.  I don't want to store an unused copy of it forever in the JavaMethod object.